lib/src/front_end/piece_factory.dart

```diff
@@ -483,7 +515,7 @@ mixin PieceFactory implements CommentWriter {
         leftBracket: leftBracket,
         elements,
         rightBracket: rightBracket,
-        style: const ListStyle(commas: Commas.nonTrailing, splitCost: 2));
+        style: const ListStyle(commas: Commas.nonTrailing, splitCost: 3));
```

Comment:
Can this change be mentioned in the commit message?

Reply:
It was a knock-on part of: "I also tweaked the cost heuristics around splitting between return types, parameter lists, and => expression bodies." But, yeah, I'll update it to mention that.